src/webui/www/public/scripts/client.js

```diff
@@ -589,6 +597,29 @@ function closeWindows() {
     MochaUI.closeAll();
 };
 
+function setupCopyEventHandler() {
+    if (clipboardEvent)
+        clipboardEvent.destroy();
+
+    clipboardEvent = new Clipboard('.copyToClipboard', {
+        text: function(trigger) {
+            var textToCopy;
+
+            if (trigger.id === "CopyName") {
```

Comment:
Done. The [formatting](https://github.com/Piccirello/qBittorrent/blob/d28e6a9a52888f01111aa1e0363358508f34163b/src/webui/www/public/scripts/client.js#L609) strays from the coding guidelines in that the case statements are indented. This is not only consistent with how most everyone writes their js, but also consistent with every other instance of `switch` in the js codebase.